frontend/public/components/_resource.scss

```diff
@@ -100,6 +100,19 @@ $height: 18px;
   margin-right: 7px;
 }
 
+.co-resource-link {
+  align-items: baseline;
+  display: flex;
+  word-break: break-all; // break-all is required for Firefox
```

Comment:
Can we apply this style only for Firefox? (Or find a way around the limitation?)

Reply:
I found a workaround that does not require `word-break: break-all;`.  🥇